pkg/common/stringset/stringset_test.go

```diff
@@ -97,3 +97,46 @@ func TestStringSet_ToSlice(t *testing.T) {
 		assert.True(t, testSet.Contains(item))
 	}
 }
+
+func TestStringSet_Intersect(t *testing.T) {
+	testSet := &stringSet{
+		m: make(map[string]bool),
+	}
+
+	testSet1 := &stringSet{
+		m: make(map[string]bool),
+	}
+
+	testItems := []string{
+		"testitem1",
+		"testitem2",
+	}
+
+	// Add testItems to testSet
+	for _, item := range testItems {
```

Comment:
use string set Add method

Reply:
I can do it but then tests will be different from previous ones.